ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryMask.java

```diff
@@ -57,11 +42,13 @@ private String getMaskedStatement(final String query) {
     }
   }
 
-  private String maskConnectProperties(final String query) {
+  private static String maskConnectProperties(final String query) {
     /*
       Failed to parse statement. Mask disallowed keys by string replacing. This is a best effort
       attempt. The assumption is that properties are in right format:
-      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")")
+      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")") i.e. it matches key wrapped in double quotes
+      equals value wrapped in double or single quotes. Key and value could be empty and there could
```

Comment:
Same for the value: it should only ever be single-quoted (if the statement is valid).